database.hh

```diff
@@ -1001,7 +1001,7 @@ public:
 private:
     future<row_locker::lock_holder> do_push_view_replica_updates(schema_ptr s, mutation m, db::timeout_clock::time_point timeout, mutation_source source,
             tracing::trace_state_ptr tr_state, reader_concurrency_semaphore& sem, const io_priority_class& io_priority, query::partition_slice::option_set custom_opts) const;
-    std::vector<view_ptr> affected_views(const schema_ptr& base, const mutation& update, gc_clock::time_point now) const;
+    std::vector<view_ptr> affected_views(const schema_ptr& base, const mutation& update) const;
```

Comment:
Why was "now" ever relevant to this function?

Reply:
It wasn't, it was only there because the previous interface of `is_satisfied_by` wanted it for regular cells, and this interface was used in `partition_key_matches`, which actually didn't provide any regular cells anyway. It would also have worked if we passed any time point, since it wasn't compared with anything.